example-raft-kv/README.md

```diff
@@ -1,25 +1,83 @@
-# Example distributed KV store built upon openraft
+# Example distributed key-value store built upon openraft.
 
-It is an example of how to build a real-world kv store with openraft.
+It is an example of how to build a real-world key-value store with `openraft`.
 Includes:
-- An in-memory `RaftStorage` implementation [store](./store).
+- An in-memory `RaftStorage` implementation [store](./store/store.rs).
 
 - A server is based on [actix-web](https://docs.rs/actix-web/4.0.0-rc.2).  
   Includes:
   - raft-internal network APIs for replication and voting.
   - Admin APIs to add nodes, change-membership etc.
   - Application APIs to write a value by key or read a value by key.
 
-- Client and `RaftNetwork`([network](./network)) are built upon [reqwest](https://docs.rs/reqwest).
+- Client and `RaftNetwork`([rpc](./network/rpc.rs)) are built upon [reqwest](https://docs.rs/reqwest).
 
 ## Run it
 
+If you want to see a simulation of 3 nodes running and sharing data, you can run the cluster demo:
+
 ```shell
-cd example-raft-kv
-cargo build
 ./test-cluster.sh
 ```
```

Comment:
Do not we need to enter the sub crate folder first?

Since it is not a member of the root workspace, `cargo build` in the top-level dir does not build this `example-raft-key-value` crate.
It may confuse some users?

Reply:
@drmingdrmer I don't think it's a problem if we gonna keep the README inside another crate, users will see the file within when accessing it from the github page. I would to maybe rename it to `run_cluster_simulation.sh`. What do you think?

<img width="866" alt="Screenshot 2022-02-03 at 10 26 20" src="https://user-images.githubusercontent.com/5521682/152324954-bc3bf437-9c24-422d-9f96-d7c75e635c09.png">

Reply:
Sounds great:DDD